org.gnome.EasyTAG.yaml

```
@@ -0,0 +1,141 @@
+id: org.gnome.EasyTAG
+runtime: org.gnome.Platform
+runtime-version: 3.38
```

```suggestion
runtime-version: '3.38'
```
this has to be a string.